src/pe_crypto.cpp

```diff
@@ -179,7 +179,7 @@ namespace libtorrent
 		if (recv_buffer.crypto_packet_finished())
 		{
 			std::vector<boost::asio::mutable_buffer> wr_buf;
-			recv_buffer.mutable_buffers(wr_buf, bytes_transferred);
```

Comment:
I believe we still need the vector here though, since this is used for outgoing data as well (which definitely needs to be able to mix in disk-buffers)